### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by sushil-mod**
# Product Page 
click on shop now or any category for product page 
- Added filter component
- Added productCard component 
- Added product Context -( just for using backend products  )
- and some minor refactor
200 out of 450 lines are just product data added in backend (product.js)
## what to check 

- ProductCard -  src/components/card/ProductCard.jsx (64 lines)
- Filters  -  src/components/filter/Filters.jsx ( 68 lines)
- ProductContext  - src/context/ProductContext.jsx (46 lines)
- Product -   src/pages/product/Product.jsx ( 33 lines)

##screenshot
storeproduc

### CODE REVIEW

File: src/context/ProductContext.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     useEffect(()=>{
+
+        (async()=>{
+
+            try {
+               
+                const [response1 , response2 ] = await Promise.all( [ axios.get("/api/products") , axios.get("/api/categories") ] ) 
+                // const [response1 , response2 ] = await axios.all( [ axios.get("/api/products") , axios.get("/api/categories") ] );
+                console.log( "response1" ,  response1.data);
+                console.log( "response2" , response2.data);
```
Comment (ozairkhan2810) [inline src/context/ProductContext.jsx]: remove comment and console

**Review by ozairkhan2810 — APPROVED** [review]
just one suggestion rest looks good

File: src/context/ProductContext.jsx (hunk 1 above)
Comment (sushil-mod) [reply src/context/ProductContext.jsx]: yeah , i forgot to remove thanks

**Review by shaksham08 — APPROVED** [review]
Code looks good just remove the commented code if not needed

**Pull request merged**